docs/spelling_wordlist.txt

```diff
@@ -1734,6 +1734,7 @@ unarchived
 unassigns
 uncomment
 uncommenting
+Uncompress
```

Comment:
Is that a right word? should we just replace uncompress with decompress?
Maybe native english speaker should review this

Reply:
Both are synonyms, i am okay with either :)  yes please suggest..

Reply:
I believe decompress is more widely used, but some services like Snowflake does use *uncompress*, so we don’t really have much choice but to add this. There’s actually one usage we can control in `airflow.utils.compression`, but it’s probably not worthwhile.

By the way, this should likely be lowercase. I think our spell checking tools now is case insensitive anyway.

Reply:
done :)